css-text-4/Overview.bs

```diff
@@ -9291,6 +9291,9 @@ Default UA Stylesheet</h2>
 
 			/* do not allow white space to collapse in textarea */
 			textarea { text-space-collapse: preserve !important; }
+
+			/* preserve character grid in preformatted text */
+			pre, code, tt { text-spacing: none; }
```

Comment:
Here are some extract from the HTML UA stylesheet:
https://html.spec.whatwg.org/multipage/rendering.html#flow-content-3 :
```css
listing, plaintext, pre, xmp {
  font-family: monospace; white-space: pre;
}
```
https://html.spec.whatwg.org/multipage/rendering.html#phrasing-content-3 :
```css
code, kbd, samp, tt { font-family: monospace; }
```
This makes me think that the selector here should be expanded a little, from `pre, code, tt` to `listing, plaintext, pre, xmp, code, kbd, smp, tt`. Some of these tags are obsolete and only in the HTML UA stylesheet for compat reasons, but `tt` is one of those, and was already included, so we should probably go with the whole set.

Reply:
tt is obsolete, but it's at least standard (part of HTML4). I added kbd and samp, but hesitant to add some of the others. Once things stabilize a bit, though, we should definitely add a PR for the HTML spec.